README.md

```diff
@@ -27,35 +29,74 @@ expirationd.start(job_name, space.id, is_expired, {
-  * `tuples_per_iteration` - number of tuples will be checked by one iteration
-  * `full_scan_time` - time required for full index scan (in seconds)
-  * `iteration_delay` - max sleep time between iterations (in seconds)
-  * `full_scan_delay` - sleep time between full scans (in seconds)
-  * `on_full_scan_start` - call function on starting full scan iteration
-    Receives `(task)` as arguments.
-  * `on_full_scan_complete` - call function on complete full scan iteration.
-    Called after `on_full_scan_success` or `on_full_scan_error`.
-    Receives `(task)` as arguments.
-  * `on_full_scan_success` - call function on success full scan iteration
-    Receives `(task)` as arguments.
-  * `on_full_scan_error` - call function on error full scan iteration
-    Receives `(task, error)` as arguments.
-  * `force` - run, even on replica
+    * `process_expired_tuple` - Applied to expired tuples, receives (space_id, args, tuple) as arguments. Can be nil: by default, tuples are removed.
+    * `index` - Name of the index to iterate on. If omitted, will use the primary index. If there's no index with this name, will throw an error. supported index types are TREE and HASH, using other types will result in an error.
+    * `iterator_type` - Type of the iterator to use, as string or box.index constant, for example, 'EQ' or box.index.EQ. Default is box.index.ALL. See https://www.tarantool.io/en/doc/latest/reference/reference_lua/box_index/pairs/.
+    * `start_key` - Start iterating from the tuple with this index value. Or when iterator is 'EQ', iterate over tuples with this index value. Must be a value of the same data type as the index field or fields, or a function which returns such value. If omitted or nil, all tuples will be checked.
+    * `tuples_per_iteration` - Number of tuples to check in one batch (iteration). Default is 1024.
+    * `atomic_iteration` - Boolean, false (default) to process each tuple as a single transaction. True to process tuples from each batch in a single transaction.
+    * `process_while` - Function to call before checking each tuple. If it returns false, the task will stop until next full scan.
+    * `iterate_with` - Function which returns an iterator object which provides tuples to check, considering the start_key, process_while and other options. There's a default function which can be overriden with this parameter.
+    * `on_full_scan_start` - Function to call before starting a full scan iteration.
+    * `on_full_scan_complete` - Function to call after completing a full scan iteration.
+    * `on_full_scan_success` - Function to call after successfully completing a full scan iteration.
+    * `on_full_scan_error` - Function to call after terminating a full scan due to an error.
+    * `args` - Passed to is_tuple_expired and process_expired_tuple() as additional context.
+    * `full_scan_time` - Time required for a full index scan (in seconds).
```

Comment:
This option is unclear -- does it allow to stop scanning after the specified period of time?

Reply:
There is a special ticket for this(https://github.com/tarantool/expirationd/issues/38). Perhaps a readme fix on this is needed by a separate PR. @Totktonada can you suggest a right way?

Reply:
To be honest, I don't get Alexey's question.

Reply:
I agree with a separate PR for the doc. But it is not directly related to #38, I think we need a more tight description, which options must be changed.

Reply:
Ticket: https://github.com/tarantool/expirationd/issues/60

Reply:
Why do you need to change the formatting for options not related to your changes?